docs/designs/mascot-poses.html

```
@@ -0,0 +1,262 @@
+  <path d="M50 24 C55 12 62 8 72 11 C66 15 60 19 57 24 Z" fill="url(#leaf)" stroke="#2B1A10" stroke-width="2" stroke-linejoin="round"/>
+  <path d="M50 26 C50 20 52 14 57 9" fill="none" stroke="#2B1A10" stroke-width="3.5" stroke-linecap="round"/>
+</g>
+<g id="cheeks"><ellipse cx="30" cy="35" rx="5" ry="2.5" fill="#FFF6EA" opacity=".5" transform="rotate(-40 30 35)"/><ellipse cx="34" cy="64" rx="6" ry="3.5" fill="#F27C9A" opacity=".7"/><ellipse cx="74" cy="63" rx="6" ry="3.5" fill="#F27C9A" opacity=".8"/></g>
+<g id="eyes-open"><circle cx="47" cy="55" r="3.5" fill="#2B1A10"/><circle cx="66" cy="54" r="4" fill="#2B1A10"/><circle cx="48" cy="54" r="1" fill="#fff"/><circle cx="67" cy="53" r="1.2" fill="#fff"/></g>
+<g id="eyes-closed"><path d="M43 55 q4 3 8 0 M62 54 q4 3 8 0" fill="none" stroke="#2B1A10" stroke-width="2.5" stroke-linecap="round"/></g>
+<g id="eyes-wide"><circle cx="47" cy="55" r="5" fill="#fff" stroke="#2B1A10" stroke-width="1.5"/><circle cx="66" cy="54" r="5.5" fill="#fff" stroke="#2B1A10" stroke-width="1.5"/><circle cx="48" cy="55" r="2.5" fill="#2B1A10"/><circle cx="67" cy="54" r="2.8" fill="#2B1A10"/></g>
+<g id="smile"><path d="M50 66 Q58 73 66 65" fill="none" stroke="#2B1A10" stroke-width="3" stroke-linecap="round"/></g>
+<g id="grin"><path d="M48 65 Q58 78 68 64 Z" fill="#2B1A10"/></g>
+<g id="o"><ellipse cx="58" cy="69" rx="4" ry="5" fill="#2B1A10"/></g>
+<g id="flat"><path d="M52 68 L64 67" fill="none" stroke="#2B1A10" stroke-width="3" stroke-linecap="round"/></g>
+<g id="body"><use href="#top" transform="translate(4 0) rotate(8 50 20)"/><circle cx="50" cy="56" r="34" fill="url(#body)" stroke="#2B1A10" stroke-width="2"/><use href="#cheeks"/></g>
```

HTMLHint (1.9.2)

[error] The id value [ body ] must be unique.

(id-unique)